tests/files/test_downloads.py

```diff
@@ -78,6 +79,41 @@ def test_file_download_url(clients, db, mocker):
     )
 
 
+def test_file_download_url_develop(clients, db, mocker, settings):
+    settings.DEVELOP = True
+    management.call_command("setup_test_user")
+    client = clients.get("Administrators")
+
+    study = StudyFactory()
+    file = FileFactory(study=study)
+    version = file.versions.latest("created_at")
+
+    mock_resp = mocker.patch("creator.files.views.HttpResponse")
+    mock_resp.return_value = HttpResponse(open("tests/data/data.csv"))
+
+    assert File.objects.count() == 1
+
+    query = "{allFiles { edges { node { downloadUrl } } } }"
+    query_data = {"query": query.strip()}
+    resp = client.post(
+        "/graphql", data=query_data, content_type="application/json"
+    )
+    file_json = resp.json()["data"]["allFiles"]["edges"][0]["node"]
+    expect_url = (
+        f"http://testserver/download/study/{study.kf_id}/file/{file.kf_id}"
+    )
+    assert resp.status_code == 200
+    assert "data" in resp.json()
+    assert "allFiles" in resp.json()["data"]
+    assert file_json["downloadUrl"] == expect_url
+    resp = client.get(file_json["downloadUrl"])
```

Comment:
You can probably remove all of the bits about downloading the actual file. This is tested in other tests above.